docs/receipt-verifier.md

```diff
@@ -0,0 +1,118 @@
+3. Update your meta tag's `content` to be this new value.
+
+For example, if your payment pointer is `$wallet.example.com/alice` and your receipt verifier service’s URL is `https://receipt-verifier.com`, then you’ll set your meta tag’s `content` to either of the following:
+
+* `https://receipt-verifier.com/%24wallet.example.com%2Falice`
+* `$receipt-verifier.com/%24wallet.example.com%2Falice`
+
+
+## Web Monetization revshare generator
+
+If you used the [Web Monetization revshare generator](https://webmonetization.org/prob-revshare) to create a meta tag for [probabilistic revenue sharing](https://webmonetization.org/docs/probabilistic-rev-sharing/), follow the same instructions as above: URI-encode your revshare payment pointer and put it in the path of the receipt verifier service’s URL.
+
+For example, if your meta tag looks like this:
+
+`<meta name="monetization" content="$webmonetization.org/api/revshare/pay/W1siJHdhbGxldC5leGFtcGxlL2FsaWNlIiwxMCwiQWxpY2UiXSxbIiR3YWxsZXQuZXhhbXBsZS9ib2IiLDEwLCJCb2IiXV0">`
+
+And your receipt verifier service’s URL is `https://receipt-verifier.com`, then you’ll set your meta tag’s `content` to either of the following:
+
+* `https://receipt-verifier.com/%24webmonetization.org%2Fapi%2Frevshare%2Fpay%2FW1siJHdhbGxldC5leGFtcGxlL2FsaWNlIiwxMCwiQWxpY2UiXSxbIiR3YWxsZXQuZXhhbXBsZS9ib2IiLDEwLCJCb2IiXV0`
+* `$receipt-verifier.com/ %24webmonetization.org%2Fapi%2Frevshare%2Fpay%2FW1siJHdhbGxldC5leGFtcGxlL2FsaWNlIiwxMCwiQWxpY2UiXSxbIiR3YWxsZXQuZXhhbXBsZS9ib2IiLDEwLCJCb2IiXV0`
+
+## Set up a monetizationprogress event listener
+
+When a WM sender receives a receipt, the sender notifies the user agent of the successful payment by submitting a `monetizationprogress` event containing a receipt.
+
+Add the following client-side code to your website to listen for a `monetizationprogress` event containing a `receipt`. When a receipt exists, your website will verify the receipt in the event listener to confirm the payment.
+
+```html
+<head>
+  <!-- This should be set to .... -->
+  <meta name="monetization" content="https://receipt-verifier.com/%24wallet.example.com%2Falice">
+  <script>
+    if (document.monetization) {
+        document.monetization.addEventListener('monetizationprogress', event => {
+            // A payment has been received
+
+            // Connect to your site’s backend to validate the payment
+            const res = await fetch('/verifyReceipt', {
+              method: 'POST',
+              headers: {
+                'Content-Type': 'application/json'
+              },
+              body: JSON.stringify({
+                receipt: event.detail.receipt
+              })
+            })
+        })
+    }
+  </script>
+</head>
+```
+
+Your backend can send the receipt to the receipt verifier’s `/verify` endpoint. Here’s an example for an Express.js server:
+
+```javascript
+app.post('/verifyReceipt', async (req, res) => {
+  const resp = await fetch('https://receipt-verifier.com/verify', {
```

Comment:
unless we own receipt-verifier.com it's better to not use a real domain in an example. `receipt-verifier.example` would be safer